src/level/LevelDirector.js

```diff
@@ -111,7 +111,7 @@
                         );
                     } else {
                         obj = me.pool.pull(
-                            "me.Entity",
+                            me.pool.exists(settings.className) ? settings.className : "me.Entity",
```

Comment:
And actually, it should just use `settings.name` to match the non-collision-layer entities.